config/jobs/ppc64le-cloud/builds/kubernetes-postsubmit.yaml

```diff
@@ -44,3 +44,94 @@ postsubmits:
+
+                mkdir -p $HOME/.terraform.d/plugins/linux_`go env GOARCH`
+
+                pushd $GOPATH/src/github.com/IBM-Cloud/terraform-provider-ibm
+                go build .
+                cp terraform-provider-ibm $HOME/.terraform.d/plugins/linux_`go env GOARCH`/terraform-provider-ibm_v1.10.0
+                popd
+
+                go get github.com/hashicorp/terraform-provider-null
+                cp $GOPATH/bin/terraform-provider-null $HOME/.terraform.d/plugins/linux_`go env GOARCH`/
+
+                go get sigs.k8s.io/kubetest2@latest
+                go get sigs.k8s.io/kubetest2/kubetest2-tester-exec@latest
+
+                TIMESTAMP=$(date +%s)
+
+                # Details of IBM S3 storage
+                S3_SERVER=s3.us-south.cloud-object-storage.appdomain.cloud
+                BUCKET=ci-builds
+                DIRECTORY=kubernetes/master/golang/master
+                build_commit=$(jq -r .commit $GOPATH/src/github.com/ppc64le-cloud/builds/kubernetes/master/golang/master/build.yaml)
+
+                K8S_BUILD_VERSION=$build_commit
+                # e2e.test binary needed for running tests using k8s tests bits from IBM s3 storage
+                URL=https://$S3_SERVER/$BUCKET/$DIRECTORY/$K8S_BUILD_VERSION
+                wget $URL/kubernetes-test-linux-ppc64le.tar.gz -P /usr/local/bin
+                tar -zxf /usr/local/bin/kubernetes-test-linux-ppc64le.tar.gz --directory /usr/local/bin
+                cp /usr/local/bin/kubernetes/test/bin/e2e.test /usr/local/bin/kubernetes/test/bin/ginkgo /usr/local/bin
+                # kubectl needed for the e2e tests
+                wget $URL/kubernetes-client-linux-ppc64le.tar.gz -P /usr/local/bin
+                tar -zxf /usr/local/bin/kubernetes-client-linux-ppc64le.tar.gz --directory /usr/local/bin
+                cp /usr/local/bin/kubernetes/client/bin/kubectl /usr/local/bin/
+                chmod +x /usr/local/bin/e2e.test /usr/local/bin/ginkgo /usr/local/bin/kubectl
+
+                kubetest2 tf --powervs-dns k8s-tests \
+                    --powervs-image-name centos-82-09232020 \
+                    --powervs-region syd --powervs-zone syd04 \
+                    --powervs-service-id 013b8948-6cf4-4fae-9b70-71ddadd4f91a \
+                    --powervs-ssh-key powercloud-bot-key \
+                    --ssh-private-key /etc/secret-volume/ssh-privatekey \
+                    --build-version $K8S_BUILD_VERSION \
+                    --s3-server $S3_SERVER --bucket $BUCKET  --directory $DIRECTORY \
+                    --cluster-name k8s-cluster-$TIMESTAMP \
+                    --up --down --auto-approve \
+                    --ignore-destroy-errors \
+                    --powervs-memory 32 \
+                    --test=exec -- /usr/local/bin/ginkgo --nodes=10 /usr/local/bin/e2e.test \
+                    -- --ginkgo.focus='\[Conformance\]' \
+                    --ginkgo.skip='\[Disruptive\]|\[Serial\]' --report-dir=$HOME \
```

Comment:
Not sure what will be the right `report-dir` to show up the results in the dashboard! check once where the artifactory folder is